PennMobile/src/main/java/com/pennapps/labs/pennmobile/api/StudentLifeRf2.kt

```diff
@@ -0,0 +1,38 @@
+import com.pennapps.labs.pennmobile.classes.LaundryPreferences
+import com.pennapps.labs.pennmobile.classes.LaundryRequest
+import com.pennapps.labs.pennmobile.classes.LaundryRoom
+import com.pennapps.labs.pennmobile.classes.LaundryRoomSimple
+import com.pennapps.labs.pennmobile.classes.LaundryUsage
+import okhttp3.ResponseBody
+import io.reactivex.Observable
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.Header
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+interface StudentLifeRf2 {
+    @GET("laundry/halls/ids")
+    suspend fun laundryRooms(): Response<List<LaundryRoomSimple>>
+
+    @GET("laundry/hall/{id}")
+    fun room(@Path("id") id: Int): Observable<LaundryRoom>
```

Comment:
Can these all be "suspend" functions instead of an Observable here? This is more of a curiosity question, fine with how it is now.